openapi/spec/openapi.json

```diff
@@ -79,6 +79,59 @@
         }
       }
     },
+    "/broadcasts/{broadcast_id}/notifications": {
+      "get": {
+        "x-beta": true,
+        "tags": ["Notification Broadcasts APIs"],
+        "operationId": "broadcasts-notifications-list",
+        "summary": "Fetch notifications by broadcast id.",
+        "description": "Fetch the notifications on a notification broadcast.",
+        "parameters": [
+          { "$ref": "#/components/parameters/x_magicbell_api_key_header" },
+          { "$ref": "#/components/parameters/x_magicbell_api_secret_header" },
+          { "$ref": "#/components/parameters/broadcast_id" },
+          { "$ref": "#/components/parameters/page_query_param" },
+          { "$ref": "#/components/parameters/per_page_query_param" }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "allOf": [
+                    { "$ref": "#/components/schemas/PaginationProps" },
+                    {
+                      "type": "object",
+                      "required": ["notifications"],
+                      "properties": {
+                        "notifications": {
+                          "type": "array",
+                          "items": {
+                            "type": "object",
+                            "properties": {
+                              "id": { "$ref":  "#/components/schemas/Notification/properties/id" },
+                              "category": { "$ref":  "#/components/schemas/Notification/properties/category" },
+                              "topic": { "$ref":  "#/components/schemas/Notification/properties/topic" },
+                              "sent_at": { "$ref":  "#/components/schemas/Notification/properties/sent_at" },
+                              "user": { "$ref":  "#/components/schemas/User" },
```

Comment:
I made a change to this, where I wrap it in a `recipient` key. I believe it's going to be helpful as we improve the backend to support more use-cases